packages/pwa-kit-dev/src/utils/script-utils.ts

```diff
@@ -255,9 +278,27 @@ export const createBundle = async ({
                     })
             )
             .then(async () => {
-                const {dependencies = {}, devDependencies = {}} = await getProjectPkg()
+                const {
+                    dependencies = {},
+                    devDependencies = {},
+                    ccExtensibility = {extends: '', overridesDir: ''}
+                } = await getProjectPkg()
+                const extendsTemplate = 'node_modules/' + ccExtensibility.extends
+
+                let cc_overrides: string[] = []
+                if (ccExtensibility.overridesDir) {
+                    const overrides_files = await walkDir(
+                        ccExtensibility.overridesDir,
+                        ccExtensibility.overridesDir
+                    )
+                    const extends_files = await walkDir(extendsTemplate, extendsTemplate)
```

Comment:
These could also potentially be parallelized for performance gains, if it's slow enough to warrant.